### ISSUE TITLE
All binaries (except Docker) broken with "Unhandled exception" error when launching version 0.19.3-linux-x64

### ISSUE THREAD

**Issue by vestyc** [slskd/slskd#1024]
I get the following when attempting to launch version 0.19.3 on debian command-line.
```
Unhandled exception. System.ArgumentOutOfRangeException: Count cannot be less than zero. (Parameter 'count')
   at System.String.Ctor(Char c, Int32 count)
   at slskd.Program.PrintLogo(String version)
   at slskd.Program.Main(String[] args)
Aborted
```

Steps to reproduce:
1. `mkdir slskd-0.19.3; cd slskd-0.19.3;`
2. `wget https://github.com/slskd/slskd/releases/download/0.19.3/slskd-0.19.3-linux-x64.zip`
3. `unzip slskd-0.19.3-linux-x64.zip`
4. `./slskd`

My setup:
- I run slskd inside tmux inside debian lxc container on proxmox host.
- Last working version I was able to run was 0.19.1. Did not try 0.19.2.

**Comment by jpdillingham** [comment]
The `arm64` build is broken on debian as well.  It's a weird error in a weird place, I'll have to do some digging.

**Comment by jpdillingham** [comment]
Starting with `--no-logo` fixes it.

```
jp@rpi3arm64:~/slskd-0.19.3$ ./slskd --version
0.19.3.0 (0.19.3.0+4e232315.4e232315e3186608ff3678d4ee7c44b05ef8b09d)
```

It's been a while since I have thought about versioning but I'm 99% sure this is unintended.

**Comment by jpdillingham** [comment]
Starting the docker container produces the following:

```
[14:35:32 INF] Version: 0.19.3.0 (0.19.3.0+4e232315)
```

Two causes:
* Version getting mangled (with a git SHA?)
* Logic to print the logo is doing math to center the logo and it is breaking because it is too long

**Comment by jpdillingham** [comment]
Fix should be out in a bit

**Comment by jpdillingham** [comment]
Reopening for visibility until I can get the build to go out.  It's failed 10 times in a row due to issues downloading frontend packages from npm.  I had this same issue a few days ago and eventually it sorted itself out.

I may need to move to a self hosted runner, the matrixed Docker build takes a lot of resources.

**Comment by jpdillingham** [comment]
Finally got a build to go through by removing the `arm/v7` target.  I'll continue to iterate there to see if I can re-enable it.

New binaries are available with release [0.19.4](https://github.com/slskd/slskd/releases/tag/0.19.4)

**Comment by vestyc** [comment]
Thanks @jpdillingham! Just tried 0.19.4 and it's running fine now.
